colossalai/inference/modeling/models/llama.py

```diff
@@ -12,6 +12,7 @@
     flash_decoding_attention,
     rotary_embedding,
 )
+from colossalai.kernel.triton.flash_decoding_utils import FDIntermTensors
```

Comment:
Maybe it's better to move the file from colossalai.kernel.triton.flash_decoding_utils to somewhere inside the inference dir. As the structure has nothing directly correlated with kernels

Reply:
Done